larq_compute_engine/tflite/kernels/bconv2d_impl.h

```diff
@@ -65,21 +65,24 @@ inline void im2col(const ConvParams& params, const RuntimeShape& input_shape,
 // The inputs post_mutiply and post_activation_bias are currently float
 // in order to accomodate for batchnorm scales
 // Later this might be changed to the int8 system of multipliers+shifts
-template <class T, class TBitpacked>
+template <typename T, typename TBitpacked, typename AccumScalar,
+          typename DstScalar>
 inline void BConv2D(const ConvParams& params, const RuntimeShape& input_shape,
```

Comment:
Same as above, I'd rather rename `T` to `SrcScalar` or `TIn` or so, now that input and output type are no longer the same.

Reply:
Done in 21aac16.